src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql

```diff
@@ -105,7 +105,13 @@ VALUES
     ('4a202b19-2cd0-4650-b4f7-b89e00762168', 'smsText'             , 'sms'                          , 'per SMS text'),
     ('ccd04c97-e688-4a3b-804b-1a1dbbb4f343', 'practiceSite'        , 'practices/sites'              , 'per practice/site'),
     ('caaa4d47-58fe-4a82-8cfd-fa90a5f7dc30', 'patients_50–150k'    , 'patients'                     , 'per patient for 50,000–149,999 patients'),
-    ('03259d19-4ccf-4bfe-8c90-884d32f351bb', 'patients_150–500k'   , 'patients'                     , 'per patient for 150,000–499,999 patients');
+    ('03259d19-4ccf-4bfe-8c90-884d32f351bb', 'patients_150–500k'   , 'patients'                     , 'per patient for 150,000–499,999 patients'),
+    ('53ed1ded-6a01-4e84-a02a-d90608948082', 'patients_1–90k'      , 'patients'                     , 'per patient for 1-89,999 patients'),
+    ('bbb6622c-fd88-459f-9015-34c8bbca76ba', 'patients_90–900k'    , 'patients'                     , 'per patient 90,000 - 899,999 patients'),
```

Comment:
Some formatting changes needed here for consistency, should be using EN dashes without spaces, e.g.:

```sql
'per patient for 1–89,999 patients'
'per patient 90,000–899,999 patients'
```

Reply:
Used EN dashes without space